Docs-3590 Update Storage Disks Articles (#1358)
truenas/documentation: content/CORE/CORETutorials/Storage/Disks/DiskReplace.md

![StoragePoolsStatusDiskFailedOptions](/images/CORE/12.0/StoragePoolsStatusDiskFailedOptions.png "Options for a Failed Disk")

> It is recommended to *offline* the disk before starting the replacement. 

Reviewer: ```suggestion
We recommend you *offline* the disk before starting the replacement. 
```

This removes the device from the pool and can prevent swap issues. To offline a disk:

Go to **Storage > Pools** screen. 

Click on the <i class="material-icons" aria-hidden="true" title="Settings">settings</i> settings icon, and then select **Pool Status** to display the list of disks in the pools.

Click the <span class="material-icons">chevron_right</span> icon for the disk you plan to remove, and then select **Offline**.

Select **Confirm** to activate the **OFFLINE** button, then click **OFFLINE**. The disk should now be offline.

{{< expand "Can I use a disk that is failing but still active?" "v" >}}
There are some situations where a disk that has not completely failed can be left online to provide additional redundancy during the replacement procedure.

> This is not recommended unless the exact condition of the failing disk is known!

Reviewer: ```suggestion
We don't recommend leaving failed disks online unless you know the exact condition of the failing disk!
```
